README.md

```diff
@@ -970,6 +971,21 @@ int main(int argc, char *argv[]) {
 }
 ```
 
+### Hidden alias
+
+It is sometimes desirable to offer an alias for an argument, but without it
+appearing it in the usage. For example, to phase out a deprecated wording of
+an argument while not breaking backwards compatible. This can be done with
+the ``ArgumentParser::add_hidden_alias_for()` method.
+
+```cpp
+argparse::ArgumentParser program("test");
+
+auto& arg = program.add_argument("--suppress");
+arg.add_hidden_alias_for("--supress"); // old mispelled alias

```

Comment:
Change to `program.add_hidden_alias_for` in the README.

Reply:
Done. Commit and PR also updated accordingly